index.js

```diff
@@ -68,7 +73,36 @@ function toggleColor(element) {
   }
 }
 
+element.addEventListener("click", function() {
+  // console.log("hi")
+  // console.log(toggleColor)
+  toggleColor(element)
+})
+
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
 
+likeButton.addEventListener("click", function() {
+  // console.log("liked")
+  const likes = document.querySelector('.likes')
+  const likesNumber = parseInt(likes.textContent.split(' ')[0]) + 1
+  // debugger
+  likes.textContent = `${likesNumber} ${likes.textContent.split(' ')[1]}`
+})
 
 /***** Deliverable 3 *****/
+const goalForm = document.querySelector('#new-goal-form')
+
+goalForm.addEventListener("submit", function(e) {
+  e.preventDefault()
+  // console.log(e)
+  const goalObj = {
+    id: player.goals[player.goals.length - 1].id + 1,
+    playerId: 1, //id of player on page?
+    link: e.target.link.value, 
+    description: e.target.description.value
+  }
+  renderGoal(goalObj)
```

Comment:
Well done! I like your solution for getting the goal ID. Once we get to working with fetch, the server will take care of making the ID for us. You can also use `player.id` to get the `playerId`, since `player` is a global variable.